### PR TITLE
Revert pagination defaults to upstream

### PR DESCRIPTION
**Pull request by avaleske**
# Pull Request Template

## Issue #192

Addresses #192 

- [x] This change addresses the issue in full

## Description

In https://github.com/bloom-housing/bloom/pull/1578 we added a new PaginationAllowsAllQueryParams params type for listings. Now we can revert the changes we made to the query params defaults in https://github.com/CityOfDetroit/bloom/pull/133

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## How Can This Be Tested/Reviewed?

Please describe the tests that you ran to verify your changes. Provide instructions so we can review. Please also list any relevant details for your test configuration

- [x] Desktop View
- [x] Mobile View

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [ ] I have assigned reviewers
- [ ] I have updated the changelog to include a description of my changes

### CODE REVIEW

**Comment by willrlin** [comment]
Maybe convert this to a draft so we don't think we're going to merge it just yet?

File: backend/core/src/application-flagged-sets/application-flagged-sets.service.ts [hunk 1]
```diff
@@ -33,10 +33,7 @@ export class ApplicationFlaggedSetsService {
   async listPaginated(queryParams: PaginatedApplicationFlaggedSetQueryParams) {
     const results = await paginate<ApplicationFlaggedSet>(
       this.afsRepository,
-      {
-        limit: queryParams.limit > 0 ? queryParams.limit : Number.MAX_SAFE_INTEGER,
-        page: queryParams.page > 0 ? queryParams.page : 1,
-      },
```
Comment (willrlin) [inline backend/core/src/application-flagged-sets/application-flagged-sets.service.ts:39]: IIUC we're relying on the default value if these params aren't present? But we're now allowing the user to request page -1 (if it's in the URL for example) by not doing validation here as well?

Comment (avaleske) [reply backend/core/src/application-flagged-sets/application-flagged-sets.service.ts:39]: The pagination library checks for negative integers here:
https://github.com/nestjsx/nestjs-typeorm-paginate/blob/238190de1d7f77af0604d6741a9b4d38cbcb665e/src/paginate.ts#L128-L129, so we don't need to do it.

Context:
Andy added this (and others like it) when we set the PaginationQueryParams defaults to `-1`, to mean "get everything".  That meant we had to convert `-1` into something the pagination library could understand here.

In https://github.com/bloom-housing/bloom/pull/1578/files#diff-06004b3008e3dd6149854cbaebc150f047c6e8bdbff2dfa0818e2a73bea323d3 I set the default back to `10` to match upstream, which means we don't need to do this conversion here anymore, so I reverted this to match upstream as well.

If we wanted we could update the pagination validators to check that the provided value is a positive integer, but that should be a different PR that we can merge upstream.

**Review by anders-schneider — APPROVED** [review]
LGTM!

**Comment by avaleske** [comment]
@willrlin merging this since @anders-schneider approved. lmk if you want further additions or clarifications re the limit validation

**Pull request merged**